backend/substrapp/tasks/utils.py

```diff
@@ -220,6 +220,87 @@ def list_files(startpath):
         logger.info(f'{startpath} does not exist.')
 
 
+def build_docker_image(dockerfile_path, image_name):
+    dockerfile_fullpath = os.path.join(dockerfile_path, 'Dockerfile')
+    current_namespace = os.getenv('NAMESPACE')
+    docker_cache = os.getenv('DOCKER_CACHE_PVC')
```

Comment:
It may be better to fetch it with kubernetes python client :)

Reply:
You would need its name in order to fetch it, or rely on something like `kubernetes.client.list_pvc()` with a filter matching a label `docker-cache`, I don't know what is the best solution for this.

Reply:
I would prefer this solution as it is less error-prone than a env variable